data/src/test/kotlin/io/timemates/backend/data/files/FileEntityMapperTest.kt

```diff
@@ -0,0 +1,42 @@
+package io.timemates.backend.data.files
+
+import com.timemates.backend.validation.createOrThrow
+import com.timemates.random.SecureRandomProvider
+import io.timemates.backend.data.files.datasource.FileEntityMapper
+import io.timemates.backend.data.files.datasource.PostgresqlFilesDataSource
+import io.timemates.backend.files.types.File
+import io.timemates.backend.files.types.value.FileId
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class FileEntityMapperTest {
+    private val random = SecureRandomProvider()
+    private val mapper = FileEntityMapper()
+
+    @Test
+    fun `toDomainMapper should map to domain file correctly`() {
+        val fileId = random.randomHash(64)
```

Comment:
put `// GIVEN` comment here. It denotes that code below is input data. it's much easier to read it then